### ISSUE TITLE
[BUG] Permission issue 

### ISSUE THREAD

**Issue by shmowell** [linuxserver/docker-sonarr#278]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Interestingly I just finished setting up Radarr seamlessly with nearly the exact same configuration as Sonarr but I'm facing permission issues with Sonarr only. The error:

`-- Path: Folder '/tv/' is not writable by user 'abc'`

However you can see from the JSON file snippets below that the user does indeed have write permission

`"Binds": [
			"/sonarr/config:/config:rw",
			"/media/will/media_server/plex-media/plex-tv:/tv:rw",
			"/media/will/media_server/downloads:/downloads:rw"
		],`

`"Type": "bind",
			"Source": "/media/will/media_server/plex-media/plex-tv",
			"Destination": "/tv",
			"Mode": "rw",
			"RW": true,
			"Propagation": "rprivate"`

### Steps To Reproduce

Sonarr's docker-compose
`sonarr:
    image: lscr.io/linuxserver/sonarr:latest
    container_name: sonarr
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Etc/CST
    volumes:
      - /sonarr/config:/config
      - /media/will/media_server/plex-media/plex-tv:/tv
      - /media/will/media_server/downloads:/downloads
    ports:
      - 8989:8989
    restart: unless-stopped`

 When comparing to Radarr there isn't much difference:
 radarr:
    image: lscr.io/linuxserver/radarr:latest
    container_name: radarr
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Etc/CST
    volumes:
      - /radarr/config:/config
      - /media/will/media_server/plex-media/plex-movies:/movies
      - /media/will/media_server/downloads:/downloads
    ports:
      - 7878:7878
    restart: unless-stopped

The issue arises when I try to select a Root Folder.

### Environment

```markdown
- OS:
- How docker service was installed:
```

### CPU architecture

x86-64

### Docker creation

```bash
sonarr:
    image: lscr.io/linuxserver/sonarr:latest
    container_name: sonarr
    environment:
      - PUID=1000
      - PGID=1000
      - TZ=Etc/CST
    volumes:
      - /sonarr/config:/config
      - /media/will/media_server/plex-media/plex-tv:/tv
      - /media/will/media_server/downloads:/downloads
    ports:
      - 8989:8989
    restart: unless-stopped
```

### Container logs

```bash
`-- Path: Folder '/tv/' is not writable by user 'abc'`
```

**Comment by KoninginSamira** [comment]
I have the exact same issue on my Ubuntu machine using Docker Desktop!

I've tried to set the owner on the host manually using `sudo chown -R 1000:1000 .` to reflect the PUID and PGID from the docker compose file, which looks fine when I check it on the host with `ls -l`. But when I check it inside the container itself, it shows `root` being the owner of all files and directories.

Then I tried setting the owner inside the container (using `sudo chown -R 1000:1000 .`), which does fix the issue somewhat, but on the host the files and directories now belong to a `user #100999`.

For now, I've managed to fix it by setting the PUID and PGID to 0, but I feel like that's only a workaround and might not be as secure (it being the id of the root user and all)
